README.md

```diff
@@ -15,11 +15,12 @@ You need to have the KVM hypervisor installed, along with a few other packages:
 - libguestfs-tools-c
 - qemu-img
 - libvirt-client
+- libosinfo-bin
 
 To install the dependencies, run:
 
 ```
-sudo dnf -y install genisoimage virt-install libguestfs-tools-c qemu-img libvirt-client wget
+sudo dnf -y install genisoimage virt-install libguestfs-tools-c qemu-img libvirt-client wget libosinfo-bin
```

Comment:
Same here.

Reply:
I'm using Ubuntu 18.04, but you are right. 
Don't remember now, how i came to that package name, but when you wrote this comment i've rechecked and i guess it should be only 
```libosinfo```

as i can see in the apt search results:
```
libosinfo-1.0-0/bionic,now 1.1.0-1 amd64 [installed,automatic]
  Library for managing information about operating systems and hypervisors

libosinfo-1.0-dev/bionic 1.1.0-1 amd64
  libosinfo development files

libosinfo-bin/bionic,now 1.1.0-1 amd64 [installed]
  Tools for querying the osinfo database via libosinfo

libosinfo-l10n/bionic,bionic 1.1.0-1 all
  Translations for libosinfo
```

Reply:
If you would like, it would be great to add an Ubuntu-specific install instruction as well that matches the same packages as Fedora.

Reply:
question is if that is ubuntu specific, because before that dependency was not there (not sure if that changed due some update in ubuntu or your work. but i can add that, but would like do more testing if it's only ubuntu related. (i'm using kvm-install-vm over a year and before there were no problem)

Reply:
I haven't checked but I think some of the packages may be named differently. The libosinfo requirement is new as of v0.10.3.

We can add Ubuntu packages in a follow-up PR as well. The package name here just needs to be updated to match Fedora package names.

Reply:
if the name of that package is "libosinfo" in Fedora as well, then it's needed to remove that "-bin" from that name. I can check it later on, but unfortunately not today :-(

Reply:
and then it would match ubuntu and fedora..

Reply:
honestly i'm now confused:
1) tried to create fedora31 got this: 
```
ERR: Unknown OS variant 'fedora31'. Please update your osinfo-db.  See https://libosinfo.org/download for more information.
```
2) tried to create debian9: (basically vm created)
```
- SSH to debian: 'ssh debian@<IP address>' or 'ssh debian@debian'
- Console at spice://localhost:5901
- DONE
```
3) tried to create fedora29: 
```
ERR: Unknown OS variant 'fedora29'. Please update your osinfo-db.  See https://libosinfo.org/download for more information.
```

I can have a look later on that, but it really confuses me now

Reply:
commited changes for README.md again for Ubuntu/Debian based distros (names of packages)